benchopt/tests/test_benchmarks/dummy_benchmark/datasets/simulated.py

```diff
@@ -12,19 +13,25 @@ class Dataset(BaseDataset):
     parameters = {
         'n_samples, n_features': [
             (100, 5000),
-            (100, 10000)]
+            (100, 10000)],  # slow to simulate big correlated design
```

Comment:
why slow ?
```
In [9]: %timeit -n 1 -r 15 gen_correlated_data(n_samples=100, n_features=10_000, rho=0.6)
91.8 ms ± 4.86 ms per loop (mean ± std. dev. of 15 runs, 1 loop each)
```

Reply:
This is old  comment from your code, I forgot to remove.